test/unit/sqs/sqs-consumer-spec.js

```diff
@@ -244,10 +248,234 @@ describe('SqsConsumer', () => {
   });
 
   describe('stop', () => {
+    beforeEach(() => {
+      consumer = new SqsConsumer({sqs: sqs}, msgBody => Promise.resolve());
+    });
+
     it('should stop polling', () => {
       consumer._running=true;
       consumer.stop();
       consumer._running.should.be.false;
     });
   });
-});
\ No newline at end of file
+
+  describe('_scheduledConsuming', () => {
+    let config,
+        messages = {
+          Messages: [{Body: "{}",ReceiptHandle: 'handle1'}]
+        };
+
+    beforeEach(() => {
+      config = {
+        defaults: {
+          consumer: {
+            scheduler: {
+              scheduled: true,
+              start: '12:00:00',
+              duration: '2 hours',
+              maxVisibilityTimeout: '10 seconds'
+            }
+          }
+        }
+      };
+    });
+
+    context('when consumer config is not present', () => {
+      before(() => {
+        delete config.defaults.consumer.scheduler;
+        consumer = new SqsConsumer({sqs: sqs}, msgBody => Promise.resolve(), {sqs: config});
+      });
+
+      it('resolves the original data', () => {
+        sqs.receiveMessage.returns({ promise: () => Promise.resolve(messages)});
+        return sqs.receiveMessage()
```

Comment:
Not sure I follow this line.  Line 290 mocks the receiveMessage method and the test is calling mocked method receiveMessage.   
- Is there any reason we need to call receiveMessage when we are just testing _scheduledConsuming ?

Shouldn't the test be only calling consumer._scheduledConsuming(messages) ?